test/fixtures/issue_comment.created.json

```diff
@@ -79,7 +79,7 @@
     "created_at": "2019-01-10T08:36:52Z",
     "updated_at": "2019-01-10T08:36:52Z",
     "author_association": "MEMBER",
-    "body": "@all-contributors please add jakebolam for code, doc and infra"
+    "body": "@all-contributors please add jakebolam for code, doc and infra\n🤖 Beep boop! This comment was added automatically by [all-contributors-auto-action](https://github.com/marketplace/actions/all-contributors-auto-action).\nNot all contributions can be detected from Git & GitHub alone. Please comment any missing contribution types this bot missed.\n...and of course, thank you for contributing! 💙"
```

Comment:
I don't quite get this change, that does not look like a comment that a user would post?

Reply:
It's a comment a bot of mine posts that hits the issue. 🙂 

Example: https://github.com/JoshuaKGoldberg/create-typescript-app/pull/429#issuecomment-1546457807

Reply:
ohhh never seen this action! Cool!